app/services/reports/base_report.rb

```diff
@@ -32,7 +32,8 @@ def report_timeout
     def transaction_with_timeout
       Db::EstablishConnection::ReadReplicaConnection.new.call do
         ActiveRecord::Base.transaction do
-          ActiveRecord::Base.connection.execute("SET LOCAL statement_timeout = #{report_timeout}")
+          quoted_timeout = ActiveRecord::Base.connection.quote(report_timeout)
+          ActiveRecord::Base.connection.execute("SET LOCAL statement_timeout = #{quoted_timeout}")
```

Comment:
lol had to make the same change in my PR https://github.com/18F/identity-idp/pull/5259/files#diff-0ce3b19d89b65f31b175259b6d8b5e5e4f9827e25b8785adc561b2799d8ea3deR35-R36

Reply:
I definitely didn't steal it 🙂